pages/team/projects/pac/schedule-status/schedules.tsx

```diff
@@ -33,24 +36,18 @@ dayjs.extend(customParseFormat);
 type TPossibleResults = "approved" | "denied";
 type TSchedulesProps = inferSSRProps<typeof getServerSideProps>;
 
-export default function Schedules({ bookings, errorCode }: TSchedulesProps) {
+export default function Schedules({ bookings: fetchedBookings }: TSchedulesProps) {
   const router = useRouter();
 
   const result = asStringOrUndefined(router.query.result) as TPossibleResults | undefined;
 
+  const [bookings, setBookings] = useState<Array<ScheduledBooking> | undefined>(fetchedBookings);
+
   const [selectedBooking, setSelectedBooking] = useState<ScheduledBooking | undefined>(undefined);
 
-  useEffect(() => {
-    if (!bookings || errorCode || bookings.length === 0) {
-      router.push({
-        pathname: "/team/projects/pac/schedule-status",
-        query: {
-          ...router.query,
-          error: errorCode,
-        },
-      });
-    }
-  }, [bookings, errorCode]); // eslint-disable-line
+  const [isActionDialogOpened, setIsActionDialogOpened] = useState<boolean>(true);
```

Comment:
```suggestion
  const [isActionDialogOpened, setIsActionDialogOpened] = useState<boolean>(false);
```
I believe this was supposed to be false?